### PR TITLE
Update to version 5.0

### PR DESCRIPTION
**Pull request by m-rossi**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Close #10, #11, #12, #13

I just updated to version to 5.0 as https://github.com/sloria/sphinx-issues/issues/162 broke sphinx for me.

### CODE REVIEW

**Comment by m-rossi** [comment]
@conda-forge-admin, please rerender

**Comment by m-rossi** [comment]
@conda-forge-admin, please restart ci

**Comment by m-rossi** [comment]
@asmeurer this is ready for review

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,37 +1,43 @@
...
   home: https://github.com/sloria/sphinx-issues
+  summary: "A Sphinx extension for linking to your project's issue tracker"
   license: MIT
+  license_family: MIT
   license_file: LICENSE
-  summary: "A Sphinx extension for linking to your project's issue tracker"
+  dev_url: https://github.com/sloria/sphinx-issues
 
 extra:
   recipe-maintainers:
```
Comment (zaneselvans) [inline recipe/meta.yaml:43]: @m-rossi We might want to try and get the admins to to add additional maintainers to this feedstock. I offered to help maintain it in early 2022 and @bjlittle offered earlier this summer but there's been no word back.

Comment (m-rossi) [reply recipe/meta.yaml:43]: Yes, I saw that issue when I started the PR. Let's wait 1-2 days for an answer and then add you & me as recipe maintainers and ping conda-forge-core.

**Comment by m-rossi** [comment]
@conda-forge-admin, please rerender

**Comment by m-rossi** [comment]
@conda-forge/core This feedstock is quite outdated and the recipe maintainers do not respond. @zaneselvans & me are offering help as recipe maintainers (and I already added us to the recipe). We are asking for a review and merge here.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,38 +1,46 @@
...
 source:
-  fn: sphinx-issues-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/s/sphinx-issues/sphinx-issues-{{ version }}.tar.gz
-  md5: 0f40881c9c95c40c6e752abce61882cc
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name.replace("-", "_") }}/{{ name.replace("-", "_") }}-{{ version }}.tar.gz
+  sha256: 192e43cf071ed7aead401cd14fd15076ecb0866238c095d672180618740c6bae
 
 build:
   noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
```
Comment (ocefpaf) [inline recipe/meta.yaml]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation 
```

**Pull request merged**